src/apps/sources/pages/ViewSourcePage.tsx

```diff
@@ -56,8 +69,11 @@ export const ViewSourcePage: React.FC<Props> = ({
     retrieveSourceAndDetails(url);
   }, [url, retrieveSourceAndDetails]);
   useEffect(() => {
-    retrieveConceptsSummary(url + "concepts/");
+    retrieveConceptsSummary(url + "concepts/", 1, 1, undefined, undefined, undefined, undefined, undefined, undefined, true);
```

Comment:
@tarun can we move the string here in a const.
why are we passing hard coded values to this function?

Reply:
@rakesh-1010 Yeah. The values are hard-coded as we only need the header of first page. So, we have hardcoded the page and limit to 1 to make the response small as we only require header